src/test/java/com/cloudbees/jenkins/plugins/bitbucket/BitbucketClientMockUtils.java

```diff
@@ -181,8 +189,10 @@ private static BitbucketPullRequestValue getPullRequest() {
         BitbucketCloudBranch branch = new BitbucketCloudBranch("my-feature-branch", null, 0);
         source.setBranch(branch);
 
-        BitbucketCloudCommit commit = new BitbucketCloudCommit();
-        commit.setHash("e851558f77c098d21af6bb8cc54a423f7cf12147");
+        BitbucketCloudAuthor author = new BitbucketCloudAuthor();
+        author.setRaw("amuniz <amuniz@mail.com");
```

Comment:
seems like a typo? your missing a `>`
```suggestion
        author.setRaw("amuniz <amuniz@mail.com>");
```

Reply:
before I start changes please complete the review so I can perform all changes all at once